xep-0353.xml

```diff
@@ -237,48 +431,123 @@
-    <example caption="Initiator Sends Finish Message"><![CDATA[
+    <p>
+      This protocol in conjunction with &xep0280; and &xep0313; or appropriate
+      groupchat protocols (like &xep0045;) allows all devices of all involved
+      parties to get synchronized about session start, rejection etc. To
+      synchronize the ending of the session, all parties MUST send a message
+      stanza containing a &lt;finish/&gt; element specifying the same ID as in
+      <link url='#accept'>Accept</link> to the bare Jid of the other party or
+      the room, respectively.
+    </p>
+    <p>
+      Letting both involved parties in two-party sessions send the &lt;finish/&gt;
+      element makes sure we have the correct state in MAM archives etc. even if
+      one client suddenly looses connectivity/power. It even makes possible for
+      a client to determine if the call is still deemed "running" by the other
+      party if it manages to recover from connectivity loss &mdash; before the
+      other party runs into a timeout and sends a &lt;finish/&gt; &mdash; to
+      recover the session or formally terminate the call (by ending the Jingle
+      session and sending a &lt;finish/&gt; message itself). See
+      <link url="#tie-break-2">Tie Breaking</link> for more infos on this and
+      similar scenarios.
+    </p>
+    <p>
+      The &lt;finish/&gt; element MUST contain a &lt;reason&gt; element as
+      defined in <link url="#reason">Reason element</link>. This SHOULD use a
+      condition of &lt;success/&gt;, except in tie-breaking scenarios
+      as defined in <link url="#tie-break-2">Tie Breaking</link>.
+    </p>
+    <example caption="Inviter Sends Finish Message"><![CDATA[
 <message from='romeo@montague.example/orchard'
          to='juliet@capulet.example'
          type='chat'>
-  <finish xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
-    <reason xmlns="urn:xmpp:jingle:1">
-      <success/>
-      <text>Success</text>
-    </reason>
+  <finish xmlns='urn:xmpp:call-message:1' id='a73sjjvkla37jfea'>
+    <reason><success/></reason>
   </finish
   <store xmlns="urn:xmpp:hints"/>
 </message>
 ]]></example>
-    <example caption="Responder Sends Finish Message"><![CDATA[
+    <example caption="Invitee Sends Finish Message"><![CDATA[
 <message from='juliet@capulet.example/phone'
          to='romeo@montague.example'
          type='chat'>
-  <finish xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
-    <reason xmlns="urn:xmpp:jingle:1">
-      <success/>
-      <text>Success</text>
-    </reason>
+  <finish xmlns='urn:xmpp:call-message:1' id='a73sjjvkla37jfea'>
+    <reason><success/></reason>
   </finish>
   <store xmlns="urn:xmpp:hints"/>
 </message>
 ]]></example>
   </section2>
+  <section2 topic="Reason element" anchor="reason">
+    <p>
+      The structure of the &lt;reason&gt; element is as follows.
```

Comment:
what value does this provide over the jingle reason?

Reply:
- Jingle is only one establishment method to be used with call invites
- Most of Jingle conditions cannot be applied to call invites.
- Jingle's reason element does not have a `<tie-break/>` condition.